src/Application/Actions/Donations/Update.php

```diff
@@ -59,22 +59,6 @@ protected function action(Request $request, Response $response, array $args): Re
             throw new DomainRecordNotFoundException('Missing donation ID');
         }
 
-        /** @var Donation $donation */
-        $this->entityManager->beginTransaction();
-
-        try {
-            $donation = $this->getDonationWithRetry($args['donationId']);
-        } catch (LockWaitTimeoutException $e) {
-            $this->logger->error("Lock Wait Timeout for donationId '{$args['donationId']}' even after retry");
-            throw $e;
-        }
-
-        if (!$donation) {
-            $this->entityManager->rollback();
```

Comment:
Not a problem since this is deleted, but I'm confused now about why we ever did rollback here, since it seems like no commands have been issued that would require rolling back.